src/stream.c

```diff
@@ -1518,6 +1524,38 @@ const char *stream_name(const struct stream *strm)
 }
 
 
+/**
+ * Get the value of the stream->cname (i.e. call->local_uri)
```

Comment:
"Get the value of the RTCP Canonical end-point identifier"
Remove the hint about `call->local_uri`